assets/javascripts/discourse/initializers/discourse-automation.js

```diff
@@ -3,8 +3,58 @@ import { ajax } from "discourse/lib/ajax";
 import { makeArray } from "discourse-common/lib/helpers";
 import { withPluginApi } from "discourse/lib/plugin-api";
 
+let _btnClickHandlers = {};
+
+function _handleEvent(event) {
+  ajax(`/append-last-checked-by/${event.currentTarget.postId}`, {
+    type: "PUT",
+  }).catch(popupAjaxError);
+}
+
 function _initializeDiscourseAutomation(api) {
+  function _cleanUp() {
```

Comment:
Does this method need to be inside `_initializeDiscourseAutomation`? It's not doing any closures